File: abc/FizzBuzz/Lambda.py
def fizzBuzz(n: int) -> str:
    word = ""

    if n % 3 == 0:
        word += "Fizz"
    if n % 5 == 0:
        word += "Buzz"

    if len(word) == 0:
        word = str(n)

    return word

def main() -> None:
    for i in range(1, 101):
        print(fizzBuzz(i))

if __name__ == "__main__":
    main()

File: abc/FizzBuzz/Lambda.rb
def fizzBuzz(n)
    word = ""

    if (n % 3).zero?
        word += "Fizz"
    end
    if (n % 5).zero?
        word += "Buzz"
    end

    if word.empty?
        word = n.to_s()
    end

    word
end

if __FILE__ == $PROGRAM_NAME
    1.upto(100) do |i|
        puts(fizzBuzz(i))
    end
end
